### PR TITLE
implemented speed changing repeatadly

### PR DESCRIPTION
**Pull request by qxe35v**
Implemented with timer; trainsystem calls user's setjoystickposition function

### CODE REVIEW

**Review by Marc8015 — APPROVED** [review]
This is ok.

File: train-system/src/main/java/hu/bme/mit/train/system/TrainSystem.java [hunk 1]
```diff
@@ -25,4 +29,15 @@ public TrainUser getUser() {
 		return user;
 	}
 
+	public void startOperation() {
+		Random r = new Random();
```
Comment (Marc8015) [inline train-system/src/main/java/hu/bme/mit/train/system/TrainSystem.java:33]: Random returns a value between 0 and 1

**Review by Marc8015 — CHANGES_REQUESTED** [review]
Please review comment

File: train-system/src/main/java/hu/bme/mit/train/system/TrainSystem.java (hunk 1 above)
Comment (qxe35v) [reply train-system/src/main/java/hu/bme/mit/train/system/TrainSystem.java:33]: true dat, ill fix it

Comment (qxe35v) [reply train-system/src/main/java/hu/bme/mit/train/system/TrainSystem.java:33]: i review'd it, your ideas make no sense

**Review by Marc8015 — APPROVED** [review]
Now it's ok

**Pull request merged**